src/instructions/tables.rs

```diff
@@ -81,6 +81,6 @@ pub fn sidt() -> DescriptorTablePointer {
 #[inline]
 pub unsafe fn load_tss(sel: SegmentSelector) {
     unsafe {
-        asm!("ltr {0:x}", in(reg) sel.0, options(nostack, preserves_flags));
+        asm!("ltr {0:x}", in(reg) sel.0, options(nomem, nostack, preserves_flags));
```

Comment:
> The only weird change is in `src/instructions/tables.rs` where we change the `asm!` flags. This was caused by a merge error in #338.

That's not a merge error, that change was introduced by #323 on purpose.

Reply:
Ohhhhh, thanks for pointing out that PR. Should we also have the flag change on 0.14?

Reply:
Fixed!

Reply:
> Ohhhhh, thanks for pointing out that PR. Should we also have the `option` change on 0.14?

https://github.com/rust-osdev/x86_64/pull/358#pullrequestreview-922303533